known-issues.rst

```diff
@@ -46,14 +58,13 @@ Older platforms
 OS X specific
 -------------
 
-New versions
-^^^^^^^^^^^^
+El Capitan
+^^^^^^^^^^
 
-- El Capitan came out after our testing period, and there are known issues
-  :jirap:`DM-3200` that will be addressed in the next release.
+*No known issues.*
 
-Older platforms
-^^^^^^^^^^^^^^^
+Older systems
+^^^^^^^^^^^^^
```

Comment:
Do we know when Apple stopped doing the `/Developer` thing? ie. did it happen before the older OS X that we can build on?